contracts/Strategy.sol

```diff
@@ -240,32 +249,29 @@ contract Strategy is BaseStrategy {
         override
         returns (address[] memory)
     {}
+
     // INTERNAL ACTIONS
 
-    function _claimRewards() internal returns (uint256) {
+    function _claimRewardsAndUnstake() internal {
         uint256 stkAaveBalance = balanceOfStkAave();
-        CooldownStatus cooldownStatus;
+        CooldownStatus _cooldownStatus;
         if (stkAaveBalance > 0) {
-            cooldownStatus = _checkCooldown(); // don't check status if we have no stkAave
+            _cooldownStatus = _checkCooldown(); // don't check status if we have no stkAave
         }
 
         // If it's the claim period claim
-        if (stkAaveBalance > 0 && cooldownStatus == CooldownStatus.Claim) {
+        if (stkAaveBalance > 0 && _cooldownStatus == CooldownStatus.Claim) {
             // redeem AAVE from stkAave
             stkAave.claimRewards(address(this), type(uint256).max);
             stkAave.redeem(address(this), stkAaveBalance);
         }
     }
 
-    function _startCooldown() internal returns (uint256) {
+    function _startCooldown() internal {
         uint256 stkAaveBalance = balanceOfStkAave();
         if (stkAaveBalance == 0) return;
 
-        CooldownStatus cooldownStatus = _checkCooldown();
-
-        if (
-            cooldownStatus == CooldownStatus.None
-        ) {
+        if (forceCooldown || _checkCooldown() == CooldownStatus.None) {
```

Comment:
small gas savings

if you expect forceCooldown to be False almost always, put it at the back so it doesn't read if the real (more frequently used) condition is true